### PR TITLE
adding a substance painter export preset file

### PR DESCRIPTION
**Pull request by Vinc3r**
link is given at the bottom of user.md

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -1,284 +1,286 @@
...
-
-![glTF SubstancePainter](glTF_SubstancePainter.png)
-
-Important is the order of `occlusion`: red channel, `roughness`: green chanel, `metallic`: blue channel. Even the glTF 2.0 property is named `metallicRoughnessTexture`, the above channel order assignment is the only valid one.
-
-Furthermore, please make sure, that the `normal` is exported for `OpenGL`. Even your render engine is using a different graphics API, this is the only valid one.
+User Documentation
+------------------
+
+### glTF 2.0 Materials
```
Comment (emackey) [inline docs/user.md]: Not sure why GitHub thinks this entire document has changed.  Did the line-endings change, or the encoding change?  It's hiding the actual diffs here.

File: docs/user.md [hunk 2]
```diff
@@ -1,284 +1,286 @@
...
+[Substance Painter](https://www.allegorithmic.com/products/substance-painter) is a 3D painting software allowing you to texture, render and share your work.  
+At point of writing, the exporter of Substance Painter does not have a preset for glTF 2.0 Metallic Roughness export. However, this can be easily configured:  
+
+![glTF SubstancePainter](glTF_SubstancePainter.png)
+
+Important is the order of `occlusion`: red channel, `roughness`: green chanel, `metallic`: blue channel. Even the glTF 2.0 property is named `metallicRoughnessTexture`, the above channel order assignment is the only valid one.
+
+Furthermore, please make sure, that the `normal` is exported for `OpenGL`. Even your render engine is using a different graphics API, this is the only valid one.
+
+You can also download our [export preset file](gltf2.spexp), and place it in *C:\Users\USER\Documents\Allegorithmic\Substance Painter\shelf\export-presets* folder.
```
Comment (emackey) [inline docs/user.md]: This file path is only for Windows.  Also, I have a Windows box with a (slightly older) Substance Painter 2 installed the default way, and this folder does not match.  It installed the presets into `C:\User\USER\Documents\Substance Painter 2`, skipping the "Allegorithmic" folder and adding the number 2 to Substance Painter.  You may need clearer directions on how users can locate this folder for themselves.

File: docs/user.md (hunk 1 above)
Comment (Vinc3r) [reply docs/user.md]: yeah I see that and been afraid of breaking something, but apparently not, and I've no idea of what changes its about.
Like you said, encoding maybe ? I'm on windows 10, and notepad++ open this file with "UTF-8 (without BOM)" encoding.

File: docs/user.md (hunk 2 above)
Comment (Vinc3r) [reply docs/user.md]: OK I will add explanation, and add folder for [Mac OS](https://docs.allegorithmic.com/documentation/display/SPDOC/Creating+export+presets) (they don't talk about linux on thier doc).

**Pull request closed without merging**